swagger/paging-special.json

```diff
@@ -0,0 +1,155 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "title": "AutoRest Special Paging Test Service",
+      "description": "Long-running Operation for AutoRest",
+      "version": "1.0.0"
+    },
+    "host": "localhost:3000",
+    "schemes": [
+      "http"
+    ],
+    "produces": [
+      "application/json"
+    ],
+    "consumes": [
+      "application/json"
+    ],
+    "paths": {
+      "/pagingSpecial/nextLinkInResponseHeaders": {
+        "get": {
+          "x-ms-pageable": { "nextLinkName": null},
```

Comment:
Would it make sense to set nextLinkName to ""x-ms-nextLink" instead of null?

Looking at the docs for `x-ms-pageable`, and as I understand it, setting nextLinkName to null would mean that there is no next operation to call. However in this scenario there is, and there is also a nextLink, it is just not in the requestBody.